### DISCUSSION TITLE
gitbook hosting stops after github integration plugins

### DISCUSSION THREAD

**Discussion by Yedla009** [GitbookIO/community#443] [0. General bugs and questions]
I am new to GitBook. If something is wrong, please correct me.

I created an organization.
Made changes on the GitBook website and hosted it.
I did this many times; changes went live almost immediately.
Now, I wanted to add a JavaScript plugin.
So, I added the GitHub plugin and synced the files to GitHub.
Added my JS plugin and synced back to GitBook.
Now, if I make changes in GitBook and try to host, it won't go live anymore.
I even deleted the GitHub plugin and tried; it won't work anymore.

new changes: 
![image](https://github.com/GitbookIO/community/assets/14228270/1f19a166-587c-4bbc-888a-e24d42484f6c)

still stuck with old changes:
![image](https://github.com/GitbookIO/community/assets/14228270/df39202a-43cf-4750-affc-8b73469585d7)

**Comment by WMGameLive** [comment]
same.
it is all fine last few days, but today I try to make some changes and it doesn't work.

**Comment by Yedla009** [comment]
I have another organization besides this test one. I uninstalled the GitHub plugin yesterday. Today, GitBook is deploying the changes (which did not work yesterday) if I make any.

Possible outcomes I see:

The GitHub plugin is causing issues in GitBook deployment.
Anything synced from GitHub will create deployment issues in GitBook.

**Comment by Yedla009** [comment]
seems pip line latency issues.
